packages/account-v2/src/constants/manualFormConstants.tsx

```diff
@@ -9,18 +9,18 @@ import ProofOfAgeIcon from '../assets/manual-upload/proof-of-age.svg';
 export type TManualDocumentTypes = typeof MANUAL_DOCUMENT_TYPES[keyof typeof MANUAL_DOCUMENT_TYPES];
 
 export const MANUAL_DOCUMENT_TYPES = Object.freeze({
-    DRIVING_LICENCE: 'driving_licence',
-    NATIONAL_IDENTITY_CARD: 'national_identity_card',
-    NIMC_SLIP: 'nimc_slip',
-    PASSPORT: 'passport',
+    driving_license: 'driving_license',
+    national_identity_card: 'national_identity_card',
+    nimc_slip: 'nimc_slip',
+    passport: 'passport',
```

Comment:
```suggestion
    drivingLicense: 'driving_license',
    nationalIdentity_card: 'national_identity_card',
    nimcSlip: 'nimc_slip',
    passport: 'passport',
```

I believe we're supposed to use `airbnb` convention which has no `snake_case` for keys in an object